lib/src/models/networks.dart

```diff
@@ -38,12 +40,14 @@ final bitcoin = new NetworkType(
     bip32: new Bip32Type(public: 0x0488b21e, private: 0x0488ade4),
     pubKeyHash: 0x00,
     scriptHash: 0x05,
-    wif: 0x80);
+    wif: 0x80,
+    opreturnSize: 256);
 
 final testnet = new NetworkType(
     messagePrefix: '\x18Bitcoin Signed Message:\n',
     bech32: 'tb',
     bip32: new Bip32Type(public: 0x043587cf, private: 0x04358394),
     pubKeyHash: 0x6f,
     scriptHash: 0xc4,
-    wif: 0xef);
+    wif: 0xef,
+    opreturnSize: 256);
```

Comment:
https://stackoverflow.com/questions/24845429/maximum-size-of-data-bitcoin-op-return-tx-can-handle

It's max 80 for BTC, peercoin has a pretty big one with 256

Reply:
fixed